bot/src/main/java/com/intern/bot/config/TelegramConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.intern.bot.config;
+
+import com.intern.bot.config.properties.BotProperties;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;
+
+@Slf4j
+@RequiredArgsConstructor
+@Configuration
+public class TelegramConfig {
+
+  private final BotProperties bot;
+
+  @Bean
+  public void connect() {
+    try {
+      TelegramBotsApi botsApi = new TelegramBotsApi(DefaultBotSession.class);
+      botsApi.registerBot(bot);
+    } catch (TelegramApiException e) {
+      log.error(e.getMessage());
```

Comment:
When you log error messages, pass the cause to the logger. It will print the full stack trace, instead of absorbing it.
`log.error("Could not initialize telegram bot API.", e)`. Printing stack trace of an exception is crucial for developer.